source/includes/aggregation/builder-dsl.rb

```diff
@@ -0,0 +1,6 @@
+criteria = Tour.where('participants.name' => 'Serenity').
+   unwind(:states).
+   group(_id: 'states', :states.add_to_set => '$states').
+   project(_id: 0, states: 1)
+
+Tour.collection.aggregate(criteria.pipeline).to_a
```

Comment:
Were you able to test this code? I only got `to_json` to work

Reply:
I took this code block from the existing docs, but `to_json` makes more sense. Changed